params/stub_params.yml

```diff
@@ -0,0 +1,4 @@
+outdir: "./results"
+reads: "./data/mock_data/*_{R1,R2}.fastq.gz"
+gbkFile: "./data/mock_data/NZ_CP029563.1_Legionella_anisa.gb"
+
```

Comment:
@Mxrcon I'm thinking that overall it's a nice pattern that you've come up with, regarding a `stub_params.yml` and `stub.conf` however, as you probably understand that these are only for dev time.

Therefore, I think that we  can simply avoid adding a `stub_params.yml` and instead add those within the `stub.conf` file.

You can refer the [`params` scope](https://www.nextflow.io/docs/latest/config.html#scope-params) to add the same content and achieve the same result.

Naturally, you'd need to update the taskfile and other projects as well.

Reply:
That's a very good point, i'll do that, i think that is very similar to nextflow/rnaseq. I'll update the taskfiles and the stub profile to all the pipelines and add them to the current pull requests.

Reply:
@abhi18av I'll add these options to the #2 PR Ok? as we're discussing profiles there i think that the creation of all profiles could be discussed there.

Reply:
Okay, I've merged that PR.

You can remove this code from this PR now.

Reply:
ok ! I'll Update it now.

Reply:
Thanks @Mxrcon , I think that we can go ahead with the merge now.